Formula/g/gql.rb

```diff
@@ -0,0 +1,23 @@
+class Gql < Formula
+  desc "Git Query language is a SQL like language to perform queries on .git files"
+  homepage "https://github.com/AmrDeveloper/GQL"
+  url "https://github.com/AmrDeveloper/GQL/archive/refs/tags/0.25.0.tar.gz"
+  sha256 "7eb75c9bb49f8b51524155b9c5d64294ac2bcf2b00812e3c39c1dd7e45aaf96e"
+  license "MIT"
+  head "https://github.com/AmrDeveloper/GQL.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  def install
+    require "fileutils"
+    system "cargo", "install", *std_cargo_args
+    mv(bin/"gitql", bin/"gql", force: true)
```

Comment:
why do you need this?

Reply:
The actual executable name installed by cargo is gitql, and this name is taken in brew so I rename it match brew formula name

Reply:
We should probably name the formula `gitql` then to match the executable name?

Reply:
Gitql name is already taken in brew

Reply:
We can call this `gitql-rust`, probably?

Reply:
So if we don't modify the binary name we can keep it gql then, same as GitHub repository name too

Reply:
Can we merge it @chenrui333?

Reply:
yeah, I feel like gitql-rust might be a better name as gitql came first.

Reply:
and we can have a conflicts with gitql rather than rename the binary.

Reply:
I would prefer not to make the name related to language that maybe change in future, so can we use the name gql (It's available) and use confilicts with gitql?